integrations/jenkins.rst

```diff
@@ -7,29 +7,29 @@
 You can use `Jenkins CI` both for:
 
 - Building and testing your project, which manages dependencies with Conan, and probably a conanfile.txt file
-- Building and testing conan binary packages for a given conan package recipe (with a conanfile.py) and uploading to a
-  conan remote (Artifactory or conan_server)
+- Building and testing conan binary packages for a given Conan package recipe (with a conanfile.py) and uploading to a
+  Conan remote (Artifactory or conan_server)
 
-There is no need for any special setup for it, just install conan and your build tools in the Jenkins machine and call
-the needed ``conan`` commands.
+There is no need for any special setup for it, just install Conan and your build tools in the Jenkins machine and call
+the needed ``Conan`` commands.
 
 
 Artifactory and Jenkins integration
 ___________________________________
 
 
 If you are using `Artifactory`_ you can take advantage of the `Jenkins Artifactory Plugin`_.
-Check `here how to install the plugin`_ and `here you can check the full documentation about the DSL`_.
+Check `here how to install the plugin`_ and `here you can check the full documentation about the DSL (Domain Specific Language)`_.
```

Comment:
This make the link reference fail. Please update the name in line 119 too

Reply:
Made the requested changes in bfe6127fc96a088de85c869e29cce78c26d813fb .